ui/app/components/clients/page/counts.hbs

```diff
@@ -5,8 +5,7 @@
 
 <div class="box is-sideless is-fullwidth is-marginless is-bottomless">
   <p class="has-bottom-margin-xl">
-    This dashboard will surface Vault client usage over time. Clients represent a user or service that has authenticated to
```

Comment:
This language was just relevant to entity/non-entity clients, so removed since it shows on all tabs